bin/regression-wally

```diff
@@ -526,23 +536,22 @@ def main():
     else:
         ImperasDVLicenseCount = 10000 # effectively no license limit for non-lockstep tests
     with Pool(processes=min(len(configs),multiprocessing.cpu_count(), ImperasDVLicenseCount)) as pool:
-       num_fail = 0
-       results = {}
-       for config in configs:
-           results[config] = pool.apply_async(run_test_case,(config,))
-       for (config,result) in results.items():
-           try:
-             num_fail+=result.get(timeout=TIMEOUT_DUR)
-           except TimeoutError:
-             pool.terminate()
```

Comment:
Are you deliberately getting rid of the `pool.terminate()` function that you added recently or is this a merge issue?

Reply:
Merge error